osu.Game/Users/Drawables/DrawableAvatar.cs

```diff
@@ -30,7 +30,10 @@ public DrawableAvatar(User user = null)
         [BackgroundDependencyLoader]
         private void load(LargeTextureStore textures)
         {
-            if (user != null && user.Id > 1)
+            string avatarUrl = user?.AvatarUrl;
+            if (avatarUrl != null)
+                Texture = textures.Get(avatarUrl);
+            else if (user != null && user.Id > 1)
```

Comment:
My next question is whether there are actually cases where we *don't* have the `AvatarUrl`. Wondering if we even want this fallthrough case.

In terms of legacy systems, we've always used the URL with the attached request string to ensure the correct version of avatars are displayed, so we may want to just prefer that in all cases. Needs a bit more investigation.

Reply:
Well, I think I should do remove that since it should be replaced null as the default guest avatar.
the only concern is deleted user from [here](https://github.com/ppy/osu-web/blob/master/app/Models/DeletedUser.php), I couldn't investigate it since I never found them